src/wallet/rpcwallet.cpp

```diff
@@ -4525,7 +4560,11 @@ UniValue z_sendmany(const UniValue& params, bool fHelp)
                         "Invalid from address: should be a taddr, zaddr, UA, or the string 'ANY_TADDR'.");
```

Comment:
```suggestion
                        "Invalid from address: should be a taddr, zaddr, Unified Address, or the string 'ANY_TADDR'.");
```

Reply:
This is out of scope for this PR; it should be handled as part of #5535.